doc/classes/Image.xml

```diff
@@ -157,6 +157,14 @@
 				Fills the image with a given [Color].
 			</description>
 		</method>
+		<method name="fill_rect">
+			<return type="void" />
+			<argument index="0" name="rect" type="Rect2" />
+			<argument index="1" name="color" type="Color" />
+			<description>
+				Fills the given [Rect2] with a given [Color].
```

Comment:
```suggestion
				Fills [code]rect[/code] with [code]color[/color].
```
or at least
```suggestion
				Fills the given [Rect2] with the given [Color].
```

Reply:
Yeah, version with `[code]`s makes more sense to me too. But again, it's based on the `fill()`'s description:
https://github.com/godotengine/godot/blob/905567a9bda93bf9482debbcc644ac32eed7d087/doc/classes/Image.xml#L156-L158
I guess it's wrong too?

Reply:
Since it's a such a small change I don't foresee any harm in including it in this PR.